Misc/NEWS.d/next/Library/2017-12-22-16-47-41.bpo-32228.waPx3q.rst

```diff
@@ -0,0 +1 @@
+Reset ``write_end`` value after ``truncate()``
```

Comment:
Instead of describing the technical solution, can you describe the issue being fixed?